fontkit.d.ts

```diff
@@ -244,6 +244,303 @@ export interface Subset {
   encodeStream(): SubsetStream;
 }
 
+/**
+ * The features is an object mapping OpenType features to a boolean
+ * enabling or disabling each. If this is an AAT font, 
+ * the OpenType feature tags are mapped to AAT features.
+ */
+export interface TypeFeatures {
```

Comment:
Rather than having `OpenTypeFeatures` and `AATFeatures` extend this interface, I think the inverse will work better:
```
export interface OpenTypeFeatures  {}

export interface AATFeatures  {}

export interface TypeFeatures extends OpenTypeFeatures, AATFeatures {
  [key: string]: boolean;
}
```